src/modules/users/users.route.ts

```diff
@@ -140,4 +143,15 @@ router.get(
   }
 );
 
+// router.post('/send', async (req, res) => {
+//   try {
+//     //await mailService.sendOrderMail('willwood98@outlook.com', req.body);
+//     await mailService.sendPrescriptionMail('willwood98@outlook.com', req.body);
+//     res.status(200).json();
+//   } catch (error: any) {
+//     console.error(error);
+//     res.status(500).json({ error: error.message });
+//   }
+// });
```

Comment:
are you going to call this endpoint from the front end? or use the function in the users.service in the createOrder/Prescription functions?

Reply:
the sendOrderMail and sendPrescriptionMail functions I mean

Reply:
I will be calling the functions directly from the user service when the order or prescription creation endpoints are triggered